t/transfer.t

```diff
@@ -52,7 +52,7 @@ $first  = ping "Ping!";
 $second = wtf;
 $third  = pong "Pong!";
 
-ok $first and $second and $third;
+ok $first && $second && $third;
```

Comment:
That's a bonus. :)

Reply:
Oh, thank you. I didn't receive that PR notification, sorry if I'm a bit late.